lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,46 @@
+module MigrationSyncHelper
+  def self.migrations_column_present?
+    ActiveRecord::Base.connection.columns("miq_regions").map(&:name).include?("migrations_ran")
+  end
+
+  def self.wait_for_remote_region_migrations(region, version, wait_time = 1)
```

Comment:
Should `region` be `provider_region`?

Reply:
Nope, I like `region` better in the context of this method because it doesn't really have anything to do with pglogical anymore. I updated the use to all be `region`